nemo/collections/asr/losses/ssl_losses/__init__.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from nemo.collections.asr.losses.pt_losses.contrastive import ContrastiveLoss
+from nemo.collections.asr.losses.ssl_losses.contrastive import ContrastiveLoss
```

Comment:
Will your pretrained model still work if this name is changed ?

Reply:
Tested, seems to still work.